src/OrchardCore.Modules/OrchardCore.Media/MediaAppContextSwitches.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace OrchardCore.Media;
+
+internal static class MediaAppContextSwitches
+{
+    private const string EnableLegacyMediaFieldGraphQLFieldsKey = "OrchardCore.Media.EnableLegacyMediaFieldGraphQLFields";
```

Comment:
This could be `public` in a `public` class, to allow people to reference it instead of relying on setting the correct string (and the release notes updated).

Reply:
Good point. Please add a comment to this, so we don't open up it in the future, since it seems like an obvious step.

Reply:
Sure, you're right. I've added a comment 👍